### PR TITLE
About modal tweaks

### PR DESCRIPTION
**Pull request by ericmatte**
Hey @sabeechen !

First of all, thanks for the kind words about my work that you put in the about modal 😄 🧙  This is appreciated.
I just wanted to add a little more tweaks/cleanup to the modal, so here it is!

Also about the rest for the next release, I've installed the staging version on my HA and tested some basic functionalities, and from what I've tested, everything seems good 👍 

### What changed?

<details open>
<summary><b>Tweaks for the about modal</b></summary>
<br>

image

- I changed the layout/alignment of the items a little.
- Now there is a link that show all the extra-contributors contributions on the project
  - Example with me: [**19 contributions**](https://github.com/sabeechen/hassio-google-drive-backup/commits?author=ericmatte)
- I've also tweak the loading to wait for all avatar images to be loaded before showing the section. Without this, it can be a little glitchy on slow network (see the after/before below).

#### After

_In these examples, I've simulated a slow network. It is quicker normally._

https://user-images.githubusercontent.com/9013072/111730758-73fe4a80-8848-11eb-92fd-18dca9450863.mp4

#### Before

https://user-images.githubusercontent.com/9013072/111730968-dfe0b300-8848-11eb-8176-fc060647b69d.mp4

---

</details>

<details open>
<summary><b>Use info_outline icon instead of block when no snapshots</b></summary>
<br>

I just think it make more sense to have an info icon instead of error in that particular case.

#### After

image

#### Before

image

---

</details>

### CODE REVIEW

File: hassio-google-drive-backup/backup/static/js/about.js [hunk 1]
```diff
@@ -1,78 +1,113 @@
...
+    contributorsLoaded = true;
   }
+
+  hideElement("contributors-loading");
 }
 
 $(document).ready(function () {
-  M.Modal.init(document.querySelector("#about_modal"));
+  M.Modal.init(document.querySelector("#about_modal"), {
+    onOpenStart: () => fetchContributors(),
```
Comment (ericmatte) [inline hassio-google-drive-backup/backup/static/js/about.js:111]: No need to create a custom function `openAbout` for this; we can directly use the callback from the Materialize modal 😄

File: hassio-google-drive-backup/backup/static/js/about.js [hunk 2]
```diff
@@ -1,78 +1,113 @@
...
+
+function showElement(id) {
+  document.getElementById(id).classList.remove("default-hidden");
+}
+
+function hideElement(id) {
+  document.getElementById(id).classList.add("default-hidden");
+}
+
+async function preloadImage(url) {
```
Comment (ericmatte) [inline hassio-google-drive-backup/backup/static/js/about.js:27]: This function is what allow us to wait for all image to be loaded before removing the loading bar.

File: hassio-google-drive-backup/backup/static/js/about.js (hunk 1 above)
Comment (sabeechen) [reply hassio-google-drive-backup/backup/static/js/about.js:111]: Hrm, I should change all the other modals to use this too.

File: hassio-google-drive-backup/backup/static/js/about.js (hunk 2 above)
Comment (sabeechen) [reply hassio-google-drive-backup/backup/static/js/about.js:27]: Oh, thats nice.  I like it.  I'll use this trick more in the future.

**Comment by sabeechen** [comment]
It looks great, I have no feedback.  Thanks for restructuring the dialog a bit, I wasn't super happy with the layout from before but I got burnt out making all the other UI changes. This formatting is much better.

**Comment by sabeechen** [comment]
A new version should show up in Home Assistant for the staging addon with your changes after the [deployment](https://github.com/sabeechen/hassio-google-drive-backup/runs/2168785017?check_suite_focus=true) finishes.

**Comment by sabeechen** [comment]
Also, good call on changing the "No snapshots" icon.  Now that its visible for first time installs its probably for the best that it doesn't make them think something is wrong.  I just got used to seeing it the way it was.

**Pull request merged**